com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraUI.Rendering.Drawers.cs

```diff
@@ -0,0 +1,226 @@
+                (serialized, owner) =>
+                {
+                    EditorGUILayout.HelpBox(Styles.disabledPostprocessing, MessageType.Warning);
+                    s_PostProcessingWarningShown = true;
+                });
+
+            private static readonly CED.IDrawer PostProcessingStopNaNsWarningDrawer = CED.Conditional(
+                (serialized, owner) => !s_PostProcessingWarningShown && IsAnyRendererHasPostProcessingEnabled(serialized, UniversalRenderPipeline.asset) && serialized.stopNaNs.boolValue,
+                (serialized, owner) =>
+                {
+                    EditorGUILayout.HelpBox(Styles.disabledPostprocessing, MessageType.Warning);
+                    s_PostProcessingWarningShown = true;
+                });
+
+            private static readonly CED.IDrawer PostProcessingDitheringWarningDrawer = CED.Conditional(
+                (serialized, owner) => !s_PostProcessingWarningShown && IsAnyRendererHasPostProcessingEnabled(serialized, UniversalRenderPipeline.asset) && serialized.dithering.boolValue,
+                (serialized, owner) =>
+                {
+                    EditorGUILayout.HelpBox(Styles.disabledPostprocessing, MessageType.Warning);
+                    s_PostProcessingWarningShown = true;
+                });
+
+            static readonly CED.IDrawer BaseCameraRenderTypeDrawer = CED.Conditional(
+                (serialized, owner) => (CameraRenderType)serialized.cameraType.intValue == CameraRenderType.Base,
+                CED.Group(
+                    Drawer_Rendering_RenderPostProcessing
+                    ),
+                PostProcessingWarningDrawer,
+                CED.Group(
+                    Drawer_Rendering_Antialiasing
+                    ),
+                PostProcessingAAWarningDrawer,
+                CED.Conditional(
+                    (serialized, owner) => !serialized.antialiasing.hasMultipleDifferentValues,
+                    CED.Group(
+                        GroupOption.Indent,
+                        CED.Conditional(
+                            (serialized, owner) => (AntialiasingMode)serialized.antialiasing.intValue ==
+                            AntialiasingMode.SubpixelMorphologicalAntiAliasing,
+                            CED.Group(
+                                Drawer_Rendering_SMAAQuality
+                            )
+                        )
+                    )
+                    ),
+                CED.Group(
+                    CameraUI.Rendering.Drawer_Rendering_StopNaNs
+                    ),
+                PostProcessingStopNaNsWarningDrawer,
+                CED.Conditional(
+                    (serialized, owner) => serialized.stopNaNs.boolValue && CoreEditorUtils.buildTargets.Contains(GraphicsDeviceType.OpenGLES2),
+                    (serialized, owner) => EditorGUILayout.HelpBox("Stop NaNs has no effect on GLES2 platforms.", MessageType.Warning)
```

Comment:
Maybe we can move this string in the Styles since it is static? (even if not localizable)

Reply:
done